src/CPUSolver.cpp

```diff
@@ -681,13 +681,13 @@ void CPUSolver::transportSweep() {
   if (_cmfd != NULL && _cmfd->isFluxUpdateOn())
     _cmfd->zeroCurrents();
 
-  /* Loop over azimuthal angle halfspaces */
-  for (int i=0; i < 2; i++) {
+  /* Loop over azimuthal angle and periodic track halfspaces */
+  for (int i=0; i < 6; i++) {
```

Comment:
I agree, it is a bit of a misnomer now. At first glance, it appears that periodic boundary conditions could be separated into their own half space, but after thinking about it for a while I convinced myself that there are situations that require 3 half spaces (or rather 3 separate parallel transport sweeps). I changed it to `_num_parallel_track_groups` in my 3D-MOC branch. Any ideas on a term that would be most fitting?